# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/scroll-animations/css/scroll-timeline-anonymous-source.html.ini
# Commit: 425cbe3c033b
# Full Hash: 425cbe3c033bfc25795979513ebd2bb9fda7338d
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2024-11-28 21:54:11
# Regressor Bug: 1933829
# File Overlap Count: 1
# Description:
#   Bug 1933829 - [wpt-sync] Update web-platform-tests to 2d5d222739fdf91743f8cbc58094976a613194e7, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 2d5d222739fdf91743f8cbc58094976a613194e7
# ==============================================================================

diff -r 732bcd7ac971 -r 425cbe3c033b testing/web-platform/meta/scroll-animations/css/scroll-timeline-anonymous-source.html.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/meta/scroll-animations/css/scroll-timeline-anonymous-source.html.ini	Wed Nov 27 22:05:56 2024 +0000
@@ -0,0 +1,9 @@
+[scroll-timeline-anonymous-source.html]
+  [CSS animation correctly uses the <html> element as the source for the default scroll() timeline]
+    expected: FAIL
+
+  [CSS animation correctly uses the <html> element as the source for the root scroll() timeline]
+    expected: FAIL
+
+  [CSS animation correctly uses the <html> element as the source for the nearest scroll() timeline]
+    expected: FAIL